.github/push-image.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Pushes a libero container image to Docker Hub with tags"
+    echo 
+    echo "Usage: $0 project input_tag"
```

Comment:
I think we have to make it clear, every time we mention a tag, that it is a Git tag or a Docker image tag